foreman/foreman.spec

```diff
@@ -196,22 +198,45 @@ BuildRequires: %{?scl_prefix}rubygem(x-editable-rails) >= 1.5.5
 BuildRequires: %{?scl_prefix}rubygem(x-editable-rails) < 1.6.0
 BuildRequires: %{?scl_prefix}rubygem(deacon) >= 1.0
 BuildRequires: %{?scl_prefix}rubygem(deacon) < 2.0
+BuildRequires: %{?scl_prefix}rubygem(webpack-rails) >= 0.9.7
+BuildRequires: %{?scl_prefix}rubygem(webpack-rails) < 1.0.0
 # assets
 %if 0%{?scl:1}
 BuildRequires: %{scl}-runtime-assets >= 3
 BuildRequires: %{scl}-runtime-assets < 4
 %endif
+BuildRequires: npm(expose-loader) >= 0.6.0
+BuildRequires: npm(expose-loader) < 0.7.0
+BuildRequires: npm(babel-preset-es2015) >= 6.6.0
+BuildRequires: npm(babel-preset-es2015) < 6.7.0
+BuildRequires: npm(babel-core) >= 6.7.2
+BuildRequires: npm(babel-core) < 6.8.0
+BuildRequires: npm(babel-loader) >= 6.2.4
+BuildRequires: npm(babel-loader) < 6.3.0
+BuildRequires: npm(stats-webpack-plugin) >= 0.2.1
+BuildRequires: npm(stats-webpack-plugin) < 1.0.0
+BuildRequires: npm(webpack) >= 1.9.11
+BuildRequires: npm(webpack) < 2.0.0
+BuildRequires: npm(jquery) >= 1.11.0
+BuildRequires: npm(jquery) < 1.12.0
+BuildRequires: npm(jquery-ujs) >= 1.2.0
+BuildRequires: npm(jquery-ujs) < 1.3.0
+BuildRequires: npm(jquery.cookie) >= 1.4.1
+BuildRequires: npm(jquery.cookie) < 1.5.0
+BuildRequires: npm(jstz) >= 1.0.7
+BuildRequires: npm(jstz) < 1.1.0
+BuildRequires: npm(lodash) >= 2.4.1
```

Comment:
This _minimum_ requirement should be wrapped in a `%if 0%{scl:1}` conditional, and ditto on the assets Require, for use on Fedora 24.

F24 contains npm(lodash) 3.x, but since this ships an old version then removing the minimum dependency will ensure the correct version gets installed (dnf/yum will try to prefer 3.x as it matches >= 2.4.1).

<pre>
%if 0%{?scl:1}
BuildRequires: npm(lodash) >= 2.4.1
%endif
BuildRequires: npm(lodash) < 2.5.0
</pre>